docs/source/user-guide/troubleshooting.rst

```diff
@@ -6,17 +6,114 @@ Troubleshooting
    :local:
    :depth: 1
 
+Numpy MKL library load failed
+=============================
+
+Error messages like
+
+    Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll
+
+or
+
+    The ordinal 241 could not be located in the the dynamic link library
+
+Cause
+-----
+
+Numpy is unable to load the correct MKL or intel openmp runtime libraries. This
+is almost always caused by one of two things:
+
+  1. the environment with numpy has not been activated
+  2. another software vendor has installed MKL or intel openmp (libiomp5md.dll)
+     files into the C:\Windows\System32 folder. These files are being loaded
+     before Anaconda's, and they're not compatible.
+
+Solution
+--------
+
+If you are not activating your environments, start with doing that. There's more
+info at the activate-env_ page. If you are still stuck, you need to consider
+more drastic measures.
+
+  1. remove any mkl-related files from C:\Windows\System32. We recommend
+     renaming them to add .bak to the filename to effectively hide them. Observe
+     if any other software breaks. Try moving the DLL files alongside the exe of
+     the software that broke. If it works again, you can keep things in the
+     moved state - Anaconda doesn't need MKL in System32, and nothing should. If
```

Comment:
"nothing should"  I don't follow what this is trying to say.  Is this no other software should need it there either?  if so, I'd recommend rewording it as "no other software should need it either"